dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityQueryParams.java

```diff
@@ -973,14 +973,14 @@ public enum OrderColumn {
         "trackedEntity",
         "uid",
         MAIN_QUERY_ALIAS), // Ordering by id is the same as ordering by created date
-    CREATED(CREATED_ID, "trackedentityinstanceid", MAIN_QUERY_ALIAS),
```

Comment:
@enricocolasante I realized that this class might also be used by old tracker 🤦🏻 I assume that `order=created:asc` in old tracker should be equivalent to `order=createdAt:asc` in new tracker, right?

Reply:
That is true.
I don't understand where is the bug and what is the fix though.
I am missing something
We are just changing the order column from `trackedentityinstanceid` to `created`, right?
Why was not working before?
Why should be working now?

Reply:
https://github.com/dhis2/dhis2-core/pull/14631/files#diff-6e6269e3c03b721125e0fee07a0b409b9aa0808e106a27f541c2c235ccef4a0aR977 is the bug

`order=createdAt` is mapped to SQL `order by trackedentityinstanceid` instead of `order by created`

Now the bug was originally only reported for new tracker by @zubaira but it also affects old tracker where 

`order=created` is mapped to SQL `order by trackedentityinstanceid` instead of `order by created`

> We are just changing the order column from trackedentityinstanceid to created, right?

No, its the other way round. `CREATED(CREATED_ID, "trackedentityinstanceid", MAIN_QUERY_ALIAS),` is order property, SQL column. Using the `CREATED_ID` which is supposed to represent the column name might have mislead you into thinking that 🤷🏻 

Does that answer your questions?

I will add a fix for old tracker in this PR as well.